jest.config.js

```diff
@@ -3,16 +3,7 @@
 // 2. Any wrong timezone handling could be hidden if we use UTC/GMT local time (which would happen in CI).
 process.env.TZ = 'Pacific/Easter'; // UTC-06:00 or UTC-05:00 depending on daylight savings
 
-const esModules = [
-  'ol',
-  'd3',
-  'd3-color',
-  'd3-interpolate',
-  'delaunator',
-  'internmap',
-  'robust-predicates',
-  'sinon',
-].join('|');
+const esModules = ['ol', 'd3', 'd3-color', 'd3-interpolate', 'delaunator', 'internmap', 'robust-predicates'].join('|');
```

Comment:
I just removed the `sinon` entry, but prettier decided to make the diff unreadable 😅